### DISCUSSION TITLE
Remnant storyline broken?

### DISCUSSION THREAD

**Discussion by DireHo** [endless-sky/endless-sky#6921] [General]
Well, i became quite baffled as i seemed to be unable to progress the remnant storyline after some point. There is nothing under current missions, and i haven't unlocked the Albatross, could only assume something broke somewhere.

I didn't remember having done mission "Remnant: Learn Sign 1", and entry is not in the save. My brain hurts a bit from digging into this unprepared, but this is a requirement for more Remnant stuff, so took that as a starting point.

These provide clues about where to go. Land on Caelian. Did it dozens of times by now.
	source "Caelian"
	destination "Aventine"

Conditions:
                has "Remnant: Void Sprites 3: done" - is done
		has "Remnant: Defense 3: done" - is done
		not "Remnant: Learn Sign 1: done" - not in save
		not "Remnant: Learn Sign 1: active" - not in save
		random < 30
		or
			has "Remnant: Key Stones: done" - is done
			has "Remnant: Key Stones (Hai): done" - not in save, but i clearly remember doing it
			has "Remnant: Key Stones (Pre-Hai) 2: done" - not in save, but i clearly remember doing it
			has "Remnant: Broken Jump Drive 2: done" - is done

Don't need to go too far, "license: Remnant" is in the save and i wouldn't have gotten it without doing the key stone missions, which are a requirement for mission "Remnant: Technology Available", so something might be off. I recall doing those in one long session, and they didn't get baked in the save? 

The first set of conditions alone should give me the mission. After many docking and undocking thinking i was just getting unlucky with the random < 30, i proceeded to delete everything after "Remnant: Learn Sign 1: active", just because. Docked and undocked multiple times, again, while jumping over to nearby systems, still nothing.   

Conditions:
                has "Remnant: Void Sprites 3: done" - done
		has "Remnant: Defense 3: done" - done
		not "Remnant: Learn Sign 1: done" - not in save
		not "Remnant: Learn Sign 1: active" - not in save

Those above, without RNG, should definitely give me the mission right on, but didn't. So i deleted more stuff and left only 

Conditions:
                has "Remnant: Void Sprites 3: done" - done
		has "Remnant: Defense 3: done" - done

Landed on Caelian and bam, got the mission. Honestly, i would be very surprised if i didn't. 

Just for the sake of it, edited the file like so, which should also give me the mission, but still didn't. 

Conditions:
                has "Remnant: Void Sprites 3: done" - done
		has "Remnant: Defense 3: done" - done
		or
			has "Remnant: Key Stones: done" - done
			has "Remnant: Key Stones (Hai): done" - not in save, but i clearly remember doing it
			has "Remnant: Key Stones (Pre-Hai) 2: done" - not in save, but i clearly remember doing it
			has "Remnant: Broken Jump Drive 2: done" - done

Maybe i'm wrong, dunno. If i am, i do apologize for wasting your time with this nonsense.

Here's the save as a reference. Using ESLauncher2 v0.8.5 Game version 0.9.15-alpha 
[x x~LEARN SIGN.txt](https://github.com/endless-sky/endless-sky/files/8857265/x.x.LEARN.SIGN.txt)

**Comment by leklachu** [comment]
I loaded up your save a couple of times, flew to Caelian, and got the sign mission pretty quick, after it offered me a rush delivery job on an earlier landing both times. To be sure, I completed, aborted and declined that job and still got the sign language mission.

Maybe the RNG just realises how patient you like to be, seeing as you got all those 349s? Wants to make you feel fulfilled by refusing to roll <30 a hundred times?

(for reference I'm also on  0.9.15, recent build from git)
